doc/tools/docgram/fullGrammar

```diff
@@ -2457,11 +2508,13 @@ eqn_ipat: [
 as_name: [
 | "as" ident
 |
+| "as" ident_or_anti
 ]
 
 by_tactic: [
 | "by" tactic_expr3
 |
+| "by" tac2expr6
```

Comment:
This is an example where it makes sense that it is not exactly the same non-terminal that follows the `by` clause: in one case it is an Ltac1 expr, whereas in the other it is an Ltac2 expr.